platform/viewer/cypress/integration/common/OHIFCornerstoneToolbar.spec.js

```diff
@@ -0,0 +1,509 @@
+          .click()
+          expectedText = 'Img: 2 2/26'; 
+        cy.get(overlaySeriesInformation)
+          .should('contain.text', expectedText);  
+
+        //Test SKIP TO LAST IMAGE button
+        cy.get('[title="Skip, to last Image"]') //Title is wrong and was reported on bug #995: https://github.com/OHIF/Viewers/issues/995
+          .click()
+        expectedText = 'Img: 27 26/26';  
+        cy.get(overlaySeriesInformation)
+          .should('contain.text', expectedText);
+
+        //Test PREVIOUS IMAGE button
+        cy.get('[title="Previous Image"]')
+          .click()
+        expectedText = 'Img: 26 25/26'; 
+        cy.get(overlaySeriesInformation)
+          .should('contain.text', expectedText);  
+
+        //Click on Cine button
+        cy.get('@cineBtn')
+        .click();
+        //Vefiry if cine control overlay is hidden
+        cy.get('@cineControls')
+          .should('not.be.visible'); 
+    });
+    
+
+    it('checks if More button will prompt a modal with secondary tools', () => {
+      cy.get('.expandableToolMenu')
+        .click();
+      //Verify if overlay is displayed
+      cy.get('.tooltip-toolbar-overlay')
+        .should('be.visible');
+
+      let iconName;
+        //Click on one of the secondary tools from the overlay
+      cy.get('.tooltip-inner > :nth-child(1)')
+        .click()
+        .then(($magnifyBtn) =>{
+          cy.wrap($magnifyBtn)
+            .should('have.class', 'active')
+            .find('svg').then(($icon)=>{
+                iconName = $icon.text();
+            })
+        })
+
+        //Check if More button is active and if it has same icon as the secondary tool selected
+        cy.get('.expandableToolMenu')
+        .click()
+        .then(($moreBtn) =>{
+          cy.wrap($moreBtn)
+            .should('have.class', 'active')
+            .find('svg').should(($icon2)=>{
+              expect($icon2.text()).to.contain(iconName)
+            })
+        })
+
+      //Verify if overlay is hidden
+      cy.get('.tooltip-toolbar-overlay')
+        .should('not.be.visible');
+    });
+
+
+    it('checks if Layout tool will multiply the number of viewports displayed', () => {
+      //Click on Layout button and verify if overlay is displayed
+      cy.get('.btn-group > .toolbar-button').as('layoutBtn')
+        .click().then(() => {
+          cy.get('.layoutChooser')
+            .should('be.visible')
+            .find('td')
+            .its('length')
+            .should('be.eq', 9);
+        })
+
+        //verify if layout has changed to 2 viewports 
+        cy.get('tbody > :nth-child(1) > :nth-child(2)').click();
+        cy.get('.viewport-container').then(($viewport) =>{
+          cy.wrap($viewport)
+            .should('have.attr', 'style', 'height: 100%; width: 50%;')//checking if the layout is correct
+            .its('length')
+            .should('be.eq', 2);
+        })
```

Comment:
I think we can assert `length` here, but the actual styling checks will break with changes that should be merged today.

Semantically:
- Select X Layout
- Verify layout length reflects selection
- Repeat a few times

Instead of style checking, we would eventually add visual checks that the layouts are consistently displayed.